Deep_Learning/Dog Breeds Classification/web-app/backend/app.py

```
@@ -0,0 +1,67 @@
+import os
+from flask import Flask, render_template,request,jsonify
```

Ruff (0.8.2)

`flask.jsonify` imported but unused

Remove unused import: `flask.jsonify`

(F401)